tanner/tests/test_session_analyzer.py

```diff
@@ -57,3 +57,28 @@ def test_create_stats(self):
         redis_mock.lpush = push_list
         stats = self.loop.run_until_complete(self.handler.create_stats(self.session, redis_mock))
         self.assertEqual(stats['possible_owners'], ['attacker'])
+
+    def test_ip_locator(self):
```

Comment:
test names should reflect the function name and status, you do not have object ip_locator, you have find_location